---
title: ""
layout: archive
sitemap: true
permalink: /
author_profile: true
---

<img src="/assets/images/IMG_1604.jpg" width="340px" alt="John Hawkins" align="right" />

# About

I just completed my Ph.D. in computational and applied mathematics at the [Institute for
Computational Engineering and Sciences (ICES)](https://www.ices.utexas.edu/) in the The University
of Texas at Austin, where I am supervised by [William H. Press](numerical.recipes) and [Ilya J.
Finkelstein](finkelsteinlab.org). I am looking for postdoctoral opportunities at the moment. <br>

I was awarded the Computational Science, Engineering, and Mathematics Fellowship for my studies at
ICES, where I obtained my Masters and Ph.D. degrees. I previously earned a B.S. in Mathematics, a
B.S.E in Mechanical Engineering, and a B.A. in Humanities at Walla Walla University in Walla Walla,
Washington.

My research is centered around developing high-throughput methods and results in molecular biology.
The gold rush in molecular biology for exponentially improved insights and methods is in full
swing, and I love being a part it.  My specialty is using computational and mathematical techniques
to extract insight from large data sets and design better experiments. I have also done a small
amount of bench work. My current research is in developing novel techniques for characterizing the
CRISPR family of proteins, with the long-term goal of helping making gene-editing techniques safe
for clinical use.  I work with a team from the Finkelstein lab to develop high-throughput methods
for looking at millions of CRISPR-DNA interactions in parallel.  This includes the Chip-Hybridized
Affinity Mapping Platform (CHAMP), which repurposes Next Generation Sequencing chips to perform
massively parallel assays on sequenced DNA, as well as new methods currently in development.  Along
the way, I realized we needed higher-performing DNA barcodes than were available with current
methods, i.e.  more barcodes using fewer bases with better error-correction properties and thus
fewer demultiplexing errors. So, I developed the current state of the art DNA barcodes from first
principles: Filled/truncated Right End Edit (FREE) barcodes. And I have also performed large-scale
genomic and transcriptomic analyses to find gene families, construct phylogenies, and perform
positive selection analysis in bats, with the goal of helping find genes and pathways which enable
bats to to harbor several deadly human viruses.  Learn more details about my research
[here](/research/).

When I'm not doing research, my favorite thing in the world is to go see the world. I love
travelling, learning languages, and meeting people. Closer to home, nothing beats weekend BBQ's
with friends in the back yard.  I also love listening to podcasts and audiobooks while going for
walks. Shout out to 99 Percent Invisible, More Perfect by RadioLab, and the Harry Potter books as
read by Stephen Fry.
